core/domain/state_domain.py

```diff
@@ -1618,9 +1618,10 @@ def get_content_ids_that_are_correctly_translated(self, language_code):
         """
         correctly_translated_content_ids = []
         for content_id, translations in self.translations_mapping.items():
-            if language_code in translations and (
-                    not translations[language_code].needs_update and
-                    translations[language_code].translation != ''):
+            if (
+            	language_code in translations and
+            	not translations[language_code].needs_update
+            ):
```

Comment:
Sorry, I made some mistake previously.
```suggestion
            if (
                language_code in translations and
                not translations[language_code].needs_update
            ):
```

Reply:
(we shouldn't use tabs)

Reply:
Committed your suggestion.